packages/design-tokens/docs/pages/how-to-migrate-to-v3.stories.mdx

```diff
@@ -31,6 +31,6 @@ Team Foundations will release a codemod or stylint plugin to assist with this mi
 
 There are two significant and breaking changes that are occurring as of the next release:
 
-* The old tokens will be removed; only `$kz-var-*` tokens will be available. Perform the above steps to prepare for this change.
+* The top level token namespace will be removed; tokens will be named `$color-purple-100`, `$spacing-md` etc, without any `kz` or `kz-var` prefix.
 
 * There may be more breaking changes, such as renaming color names (e.g. Wisteria -> Purple). Follow #updates_design_systems on Slack to stay informed.
```

Comment:
```suggestion
* The names of the color tokens will use the common name. For example, Wisteria becomes Purple, Cluny becomes Blue and so on. 
* This is not an exhaustive list of changes, there may be other breaking changes. Follow #updates_design_systems on Slack to stay informed.
```

Reply:
It might be because it's getting late, but this phrasing isn't the best. Now that we know the what all the colours are changing to we should call it out specifically.